shape_producer/estimation_methods_2016.py

```diff
@@ -185,6 +183,167 @@ def get_cuts(self):
                 "ztt_genmatch_ll"))
 
 
+class ZTTEmbeddedEstimation(EstimationMethod):
```

Comment:
ich würde direkt in die Estimation '2016' rein schreiben - für 2017 sind die Faktoren ja andere

Reply:
Es ist ja in den 2016er estimation methods drin, das passt doch, oder?

Reply:
ich lasse es mal ohne jahreszahl